tests/testsuite/messages.rs

```diff
@@ -48,6 +47,14 @@ pub fn raw_rustc_output(project: &Project, path: &str, extra: &[&str]) -> String
     result
 }
 
+fn redact_rustc_message(msg: &str) -> impl IntoData {
+    use snapbox::filter::{Filter, FilterPaths};
+    let assert = assert_e2e();
+    let redactions = assert.redactions();
+    let msg = redactions.redact(msg);
+    FilterPaths.filter(msg.into())
+}
```

Comment:
And of course, we can just rewrite the assertions in this file using `assert_e2e` if needed.